src/web-router/dispatchers/security/signin-request.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios')
+const qs = require('qs')
+import { CLIENT_ID, CLIENT_SECRET } from './constants'
+import { getServiceUrl } from '../../../init-nmos'
+
+export default (username, password) => {
+  let scope = 'is05'
+  let data = {
+    grant_type: 'password',
+    scope: scope,
+    username: username,
+    password: password
+  }
+
+  return getServiceUrl('nmos-auth', 'v1.0', 105)
```

Comment:
Remove magic number
Document behaviour with auth server priority in query string when registry and auth server are on differing priorities in a README